tcms/static/style/patternfly_override.css

```diff
@@ -71,3 +71,10 @@ img {
 .property-value a {
     color: #c00; inherit !important;
 }
+
+a.disabled {
+    color: gray;
+    cursor: default;
+    pointer-events: none;
+    text-decoration: none;
+}
```

Comment:
While this class changes the buttons visually it doesn't prevent me from clicking on them. I can still click on the plus button and get a popup opened.

Reply:
Try https://learn.jquery.com/using-jquery-core/faq/how-do-i-disable-enable-a-form-element/ (also try `attr` if `prop` doesn't work).

Reply:
This is strange because they are disabled on Chrome. What kind of browser are you using?

Reply:
> Try https://learn.jquery.com/using-jquery-core/faq/how-do-i-disable-enable-a-form-element/ (also try `attr` if `prop` doesn't work).

 `prop` does not work for links also `attr` removes href,  which requires to be constructed afterwards when there is a valid pop-up link.

Reply:
> What kind of browser are you using?

Firefox.

Reply:
I have tested on "Chrome", "Firefox" and "Opera" and `+` gets disabled on "New Bug", "New Test Case", "New Test Plan" pages when there is no parent item.

Need some help here to reproduce.

Do you also expect it to be disabled on "admin" page?

Reply:
Must have been stale CSS/JS. Seems to work fine for me now.